test/test_model.py

```diff
@@ -61,7 +25,9 @@ class DummyModel(Model):
 
     # configure the code directories
     artifact_dirs = ["src1", "src2"]
-    with temporary_dirs(artifact_dirs), temporary_artifact_env(artifact_dirs):
+    with temporary_dirs(artifact_dirs), temporary_env(
```

Comment:
Could you explain the reason for renaming the function from temporary_artifact_env to temporary_env?
Also, could you explain why it was moved to a separate module as a file named util?

Reply:
it's not renaming the function def, It's replacing the function.
`temporary_artifact_env` was for setting env var specifically for artifact.
`temporary_env` is for setting env var in general not specific to artifact.
now with `temporary_env`, It can cover `temporary_artifact_env` by
`temporary_env({"ARTIFACT_DIRS": "src,code"})`
it equlas to `temporary_artifact_env("src,code")`